### PR TITLE
Use IntelilJ 2023.3

### PR DESCRIPTION
**Pull request by turkeylurkey**
Pull in Vaisakh's changes.

One change, remove the code that runs LibertyExplorer on a background thread because it caused an exception involving the need to run the code on the event thread. Also when running the tests use an option not to generate an assertion (red warning dialog) when the slow operation on event thread exception is generated.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -137,7 +137,7 @@ runIde {
 
 intellij {
     // For a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
-    version = '2023.2.3'
+    version = '2023.3'
```
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply build.gradle:140]: Don't we need to worry about people using 2023.3(.0)?

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java [hunk 2]
```diff
@@ -202,9 +202,10 @@ private static boolean containerVersion(String version){
      * @throws LibertyException
      */
     public static String getMavenSettingsCmd(Project project, VirtualFile buildFile) throws LibertyException {
+        String wrappedMaven = "Use Maven wrapper";
```
Comment (mrglavas) [inline src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java:205]: Was `MavenServerManager.WRAPPED_MAVEN` removed? Does this have the same value? Perhaps make this a static final constant.

File: build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply build.gradle]: Changed the pull request to `2023.3.4`

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java (hunk 2 above)
Comment (turkeylurkey) [reply src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java]: Made static final.

**Review by TrevCraw — APPROVED** [review]

**Review by mrglavas — APPROVED** [review]

**Pull request merged**